src/tests/ftest/util/osa_utils.py

```diff
@@ -129,3 +147,52 @@ def verify_single_object(self):
                                       "akey {0}".format(akey)))
         self.obj.close()
         self.container.close()
+
+    def ior_thread(self, pool, oclass, api, test, flags, results):
+        """Start threads and wait until all threads are finished.
+
+        Args:
+            pool (object): pool handle
+            oclass (str): IOR object class
+            api (str): IOR api
+            test (list): IOR test sequence
+            flags (str): IOR flags
+            results (queue): queue for returning thread results
+
+        """
+        processes = self.params.get("slots", "/run/ior/clientslots/*")
```

Comment:
minor: Can we have it consistent with IorTestBase class.
processes = self.params.get("np", '/run/ior/client_processes/*')

Reply:
Sure. I can do the changes.

Reply:
Done.